roc/log.go

```diff
@@ -183,6 +187,23 @@ func loggerRoutine() {
 	}
 }
 
+func logWrite(level LogLevel, text ...interface{}) {
+	if level <= loggerLevel {
```

Comment:
This is a data race because you're reading loggerLevel in logWrite and writing it in SetLogLevel concurrently without protections.

As I already wrote, you can make loggerLevel a plain integer, but you still need to use appropriatre atomic operations to work with it.